tests/sessions_tests/tests.py

```diff
@@ -424,6 +424,28 @@ def test_clearsessions_command(self):
         # ... and one is deleted.
         self.assertEqual(1, self.model.objects.count())
 
+    def test_ticket_21608(self):
+        # Test for #21608 - Logged out sessions are resurrected by concurrent requests.
+
+        # Create new session.
+        SESSION_KEY = 'tkniiqcv0js5reo8y0ofrywrhmwnt9xw'
+        s1 = DatabaseSession(SESSION_KEY)
+        s1.save(must_create=True)
+
+        # Logout in an other context.
+        s2 = DatabaseSession(SESSION_KEY)
+        s2.delete()
+
+        # Modify session in first context.
+        s1.modified = True
+        try:
+            s1.save()  # This should throw an exception as the session is deleted, not resurrect it.
+        except:
+            pass
```

Comment:
The exception thrown here is a side effect of how save(force_update=True) works. The thing the test is suppose is that the session is not inserted in the database again. Perhaps it's that inline comment that is misleading?